.github/ISSUE_TEMPLATE/DesignDoc.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Design Doc tracking issue
+about: A standard template to follow when leading a project
+labels: Daily, NewFeature
+---
+
+## Proposal
+
+## Tasks
+
+- [ ] Post **Proposal (Problem/Solution)** in #expensify-open-source
+- [ ] Wait at least one full business day, and until the post has a majority (2/3) of positive reactions (👍)
+- [ ] Paste Proposal in the space above with a link to the Slack thread
+- [ ] Email `strategy@expensify.com` and paste in the Proposal
+- [ ] Fill out **High-level overview of the problem**, **Timeline**, and **Terminology** sections of the Design Doc
+- [ ] Email `strategy@expensify.com` (same email chain as before) with the link to your Design Doc
+- [ ] Host a **pre-design** meeting ([example](https://expensify.slack.com/archives/C01GTK53T8Q/p1665794669539419)) in `#expensify-open-source` to discuss any necessary details in public before filling out the **High-level of proposed solution** section.
+- [ ] Fill out the **High-level of proposed solution** section
+- [ ] Email `stategy@expensify.com` again with links to the doc and pre-design conversation in Slack
+- [ ] Add the `DesignDocReview` label to get the **High-level of proposed solution** section reviewed
+- [ ] Respond to any questions or concerns and bring up blockers in Slack to get a consensus if necessary
+- [ ] Confirm that the doc has the minimum necessary number of reviews before proceeding
+- [ ] Host another pre-design meeting in `#expensify-open-source` to ask for engineering feedback on the technical solution.
+- [ ] Fill out the **Detailed implementation of the solution** and related sections.
```

Comment:
NAB - maybe add `.` to the end of all tasks or remove it from the 2 tasks that end with them